composer.json

```diff
@@ -9,8 +9,8 @@
 	],
 	"require": {
 		"php": "^7.4|^8.0",
```

Comment:
What about php version `^8.2`?
Also, I suppose that `tectonic/localisation` should be upgraded too and bumped here

Reply:
Opened a [PR](https://github.com/tectonic/laravel-localisation/pull/13) for this